src/scaladoc/scala/tools/nsc/doc/model/MemberLookup.scala

```diff
@@ -100,11 +100,35 @@ trait MemberLookup extends base.MemberLookupBase {
   private def isJDK(sym: Symbol) =
     sym.associatedFile.underlyingSource.map(f => isChildOf(f, (sys.props("java.home")))).getOrElse(false)
 
-  def jdkUrl(path: String): String = {
-    if (path.endsWith(".jmod")) {
+  // ISSUE-12820
+  import scala.util.Try
+  private lazy val classGetModule = Try {
+    val clazz = Class.forName("java.lang.Class")
+    clazz.getMethod("getModule")
+  }.toOption
+  private lazy val moduleGetName = Try {
+    val clazz = Class.forName("java.lang.Module")
+    clazz.getMethod("getName")
+  }.toOption
+
+  def jdkUrl(path: String, sym: Symbol): String = {
+    if (path.endsWith(".jmod") && javaVersion > 11) {
```

Comment:
maybe I'm missing something obvious, but why `> 11` and not `>= 11` (or even `>= 9`)? Wasn't this stuff added in JDK 9?

I don't actually care about JDK 9 or 10, but I am wondering why 11 is excluded here.

Reply:
Oh my goodness, very embarrassing for me, it should be `>= 11`. I sneak it into one of commit messages, but javadocs _appear_ to start using the module name as part of the path starting in 11 even though modules are added in 9. I can't find a formal announcement of this change, just something I noticed while testing. Though it appears I didn't test enough. Compare links below ...

```
https://docs.oracle.com/en/java/javase/12/docs/api/java.base/java/lang/Integer.html
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Integer.html
https://docs.oracle.com/javase/10/docs/api/java/lang/Integer.html
https://docs.oracle.com/javase/9/docs/api/java/lang/Integer.html
https://docs.oracle.com/javase/8/docs/api/java/lang/Integer.html
```

Reply:
Changed the kind of equality used in the latest force push.

Reply:
(pats self on back and says "good eye" 😁 )